src/host_vm.rs

```diff
@@ -167,15 +167,15 @@ impl<T: GuestStagePagingMode> HostVmLoader<T> {
         mut page_alloc: HypPageAlloc,
     ) -> Self {
         // Reserve a contiguous chunk for the host's FDT. We assume it will be no bigger than the
-        // size of the hypervisor's FDT and we align it to `T::TOP_LEVEL_ALIGN` to maintain the
-        // contiguous mapping guarantee from GPA -> HPA mentioned above.
+        // size of the hypervisor's FDT and we align it to 2M to maintain the contiguous mapping
```

Comment:
the `2M` should be a `const` somewhere instead of hard-coding `PageSize::Size2M` everywhere.